src/DataCollection.UWP/DataCollection.UWP.csproj

```diff
@@ -11,13 +11,20 @@
     <AssemblyName>DataCollection.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
```

Comment:
Why this? If you rely on 17763, make sure those APIs you need for that are guarded against if running on 16299